tests/helpers.py

```diff
@@ -188,7 +187,7 @@ def reset_poetry(self) -> None:
 
 class TestLocker(Locker):
     def __init__(self, lock: str | Path, local_config: dict) -> None:
-        self._lock = TOMLFile(lock)
+        self._lock = Path(lock)
```

Comment:
This change was required, because there is no reason to keep `TOMFile` when `Locker._lock` is actually a `Path`.

No test is relying on `_lock` be a `TOMLFile`.